src/main/java/com/iexec/worker/feign/client/SmsClient.java

```diff
@@ -1,26 +1,29 @@
 package com.iexec.worker.feign.client;
 
 
-import com.iexec.common.sms.SmsRequest;
+import com.iexec.common.chain.ContributionAuthorization;
 import com.iexec.common.sms.secrets.SmsSecretResponse;
 import com.iexec.worker.feign.config.FeignConfiguration;
 
-import feign.FeignException;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
 
+import feign.FeignException;
 
 @FeignClient(name = "SmsClient",
         url = "#{publicConfigurationService.smsURL}",
         configuration = FeignConfiguration.class)
 public interface SmsClient {
 
     @PostMapping("/untee/secrets")
-    ResponseEntity<SmsSecretResponse> getUnTeeSecrets(@RequestBody SmsRequest smsRequest) throws FeignException;
+    ResponseEntity<SmsSecretResponse> getUnTeeSecrets(@RequestHeader("Authorization") String authorization,
```

Comment:
Note for future: Probably rename SmsSecretResponse to something like UnteeSecretResponse (We need to talk about untee worflow anyway)

Reply:
I agree. I didn't touch that part of the workflow until we fix our requirements, thanks!